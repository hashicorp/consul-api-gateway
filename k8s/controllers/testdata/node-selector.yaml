apiVersion: gateway.networking.k8s.io/v1alpha2
kind: Gateway
metadata:
  name: node-selector
  annotations:
    "polar.hashicorp.com/node-selector": >
      {"ingress-ready":"true"}
    "polar.hashicorp.com/service-type": "NodePort"
    "polar.hashicorp.com/consul-http-address": "host.docker.internal:443"
    "polar.hashicorp.com/image": "polar:1"
    "polar.hashicorp.com/auth-method": "polar"
    "polar.hashicorp.com/service-account": "polar"
    "polar.hashicorp.com/envoy": "polar:1"
spec:
  gatewayClassName: polar
  listeners:
  - protocol: HTTP
    port: 8080
    name: http
    allowedRoutes:
      namespaces:
        from: Same
  - protocol: HTTPS
    port: 8443
    name: https
    allowedRoutes:
      namespaces:
        from: Same