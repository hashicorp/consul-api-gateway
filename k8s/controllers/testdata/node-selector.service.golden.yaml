metadata:
  creationTimestamp: null
  labels:
    managedBy: polar
    name: polar-node-selector
  name: node-selector
spec:
  ports:
  - name: http
    port: 8080
    protocol: TCP
    targetPort: 0
  - name: https
    port: 8443
    protocol: TCP
    targetPort: 0
  selector:
    managedBy: polar
    name: polar-node-selector
  type: NodePort
status:
  loadBalancer: {}
